README.md

```
@@ -76,9 +76,46 @@ npm run dev          # Servidor de desenvolvimento
+npm run test:watch   # Rodar testes em watch mode
+npm run test:coverage # Gerar relatório de cobertura
 npx prisma studio    # Abrir Prisma Studio (GUI do banco)
 ```
 
+## Testes
+
+O projeto utiliza Jest e React Testing Library para testes automatizados.
+
+### Estrutura de Testes
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)